pkgs/applications/radio/cqrlog/default.nix

```diff
@@ -0,0 +1,100 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fpc
+, lazarus
+, atk
+, cairo
+, gdk-pixbuf
+, glib
+, gtk2-x11
+, libX11
+, pango
+
+, hamlib
```

Comment:
```suggestion
, pango
, hamlib
```

Reply:
Does the presence (or absence) of newlines really matter?

Reply:
This is done to separate buildInputs from propagatedBuildInputs.

Reply:
Over time this will bitrot anyway and be confined.

Reply:
Fixed.